doc/data_format.rst

```diff
@@ -1672,73 +1672,83 @@ Gui Sprites
+ 160        4      8-     light_rog_orange          Orange of a red/orange/green light.
+ 164        4      8-     light_rog_green           Green of a red/orange/green light.
+ 168        4      8-     light_rog_none            None of a red/orange/green light.
+ 172        4      8-     light_rg_red              Red of a red/green light.
+ 176        4      8-     light_rg_green            Green of a red/green light.
+ 180        4      8-     light_rg_none             None of a red/green light.
+ 184        4      2-     pos_2d                    Flat rotation positive direction (counter clock wise).
+ 188        4      2-     neg_2d                    Flat rotation negative direction (clock wise).
+ 192        4      2-     pos_3d                    Diametric rotation positive direction (counter clock wise).
+ 196        4      2-     neg_3d                    Diametric rotation negative direction (clock wise).
+ 200        4      3-     close_button              Close Button.
+  --        4      3-3                              Maximise button.
+  --        4      3-3                              Minimise button.
+ 204        4      4-     terraform_dot             Terraform dot.
+ 208        4      9-     message_goto              Inbox: Go To Location button.
+ 212        4      9-     message_park              Inbox: Park Management button.
+ 216        4      9-     message_guest             Inbox: Guest Window button.
+ 220        4      9-     message_ride              Inbox: Ride Instance Window button.
+ 224        4      9-     message_ride_type         Inbox: Ride Select GUI button.
+ 228        4     10-     mainmenu_splash_duration  Main menu screen: Duration of the splash screen animation in milliseconds.
+ 232        4     10-     mainmenu_logo             Main menu screen: FreeRCT logo.
+ 236        4     10-     mainmenu_splash           Main menu screen: FreeRCT splashscreen.
+ 240        4     10-     mainmenu_new              Main menu screen: New Game button.
+ 244        4     10-     mainmenu_load             Main menu screen: Load Game button.
+ 248        4     10-     mainmenu_settings         Main menu screen: Settings button.
+ 252        4     10-     mainmenu_quit             Main menu screen: Quit button.
+ 256        4     10-     mainmenu_frames           Main menu screen: Number f of frames in the background animation.
+ 260        4     10-     mainmenu_duration         Main menu screen: Duration of each frame in the background animation in milliseconds.
+ 264      4*f     10-     mainmenu_frame_f          The image for each frame of the main menu's background animation.
+264+4*f     4      2-     texts                     Text of the guis (reference to a TEXT block).
+268+4*f                                             Total length.
+=======  ======  =======  ========================  ==========================================================================================
```

Comment:
I wonder if it would be useful to split the main menu items from regular gui sprites, and the former their own block.. While both are guis sprites, the regular sprites are all very small and composed to construct the gui elements. The main menu items are simply big colourful sprites you can click on.

What do you think?